include/pybind11/operators.h

```diff
@@ -157,7 +157,7 @@ PYBIND11_UNARY_OPERATOR(float,      float_,       (double) l)
 #undef PYBIND11_BINARY_OPERATOR
 #undef PYBIND11_INPLACE_OPERATOR
 #undef PYBIND11_UNARY_OPERATOR
-NAMESPACE_END(detail)
+NAMESPACE_END(detail);
```

Comment:
Accident?
If not, could you add please a comment why the semicolon is needed here?

Reply:
This removed one sphinx+breathe error; Doxygen isn't guessing the correct nature of NAMESPACE_END, and I wasn't able to get it to expand this. Finally gave up and reverted to and old Sphinx+Breathe, which works (or at least doesn't spit out warning then error out).

Reply:
(I will revert these changes after I get a new PyPy working or find a way to get the old one to download correctly)